tests/replication/tcp/test_redis.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
I think this file can get removed?

Reply:
urr, yes. I'm not sure how this came about, oops :)